src/acquire-device-kit/device/kit/camera.h

```diff
@@ -9,25 +9,46 @@ extern "C"
 {
 #endif
 
+    /// @brief Represents and allows control of a camera device.
     struct Camera
     {
         struct Device device;
         enum DeviceState state;
 
+        /// @brief Attempts to set the given properties on the given camera.
+        /// @details Setting properties can partially succeed depending on
+        ///          other property values or the state of the camera.
+        ///          If you want to be certain that a specific property was
+        ///          actually used, you should call get afterwards to check
+        ///          that property value.
         enum DeviceStatusCode (*set)(struct Camera*,
                                      struct CameraProperties* settings);
+
+        /// @brief Fills out the given properties from the given camera.
         enum DeviceStatusCode (*get)(const struct Camera*,
                                      struct CameraProperties* settings);
+
+        /// @brief Fills out the given property metadata from the given camera.
+        /// @details These metadata or capabilities may depend on the camera's
+        ///          current property values.
         enum DeviceStatusCode (*get_meta)(const struct Camera*,
                                           struct CameraPropertyMetadata* meta);
+
+        /// @brief The shape of the next frame that the camera will capture.
         enum DeviceStatusCode (*get_shape)(const struct Camera*,
                                            struct ImageShape* shape);
+        /// @brief Starts acquiring frames.
         enum DeviceStatusCode (*start)(struct Camera*);
+
+        /// @brief Stops acquiring frames.
+        /// @details This is not guaranteed to wait or block until any pending
+        ///          frames have been acquired or the camera is actually stopped.

```

Comment:
I think I wrote this because Spinnaker gave no solid guarantees about its behavior for `ICameraBase::EndAcquisition`, which was surprising/unsettling to me, so I reflected the uncertainty here. We could just omit this. Or if we think this should wait/block, then we should ensure that all implementations (including Spinnaker) do so if possible.

Reply:
I updated the details here to reflect the comment directly above.